### DISCUSSION TITLE
Package can't be installed with spatie/laravel-sitemap

### DISCUSSION THREAD

**Discussion by ibrunotome** [spatie/laravel-export#69] [Q&A]
spatie/laravel-sitemap requires `"spatie/crawler": "^5.0"` and this package require `"spatie/crawler": "^4.3"`. Could this package require `"spatie/crawler": "^5.0"` also or this will need a lot of refactor?

**Comment by scrlkx** [comment]
I think that we can make it work, depends only of a `spatie/crawler` change: https://github.com/spatie/crawler/pull/371.

EDIT: solving at https://github.com/spatie/laravel-export/pull/72
